laravel/the-basics/customizing-deployments.html.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Customizing Deployments
+layout: framework_docs
+objective: Customize how Fly deploys your Laravel application.
+order: 6
+---
+
+The `fly launch` command sets you up with a useful starting point, but you can customize just about everything to meet your needs. Here's a few things that might be useful to know.
+
+## User Scripts
+
+The `fly launch` command generated a `.fly/scripts` directory. Any file in here ending in `.sh` will run (via `bash`) anytime your application is started.
+
+User scripts are run from the `ENTRYPOINT` script before anything else is ran.
```

Comment:
This `ENTRYPOINT` script is docker/entrypoint.sh right? would be great to reference the location here if so

Reply:
Nope, you were right - I'll reference that! `.fly/entrypoint.sh` is the entrypoint script (which is saved to `/entrypoint` within the Docker image that is built)